airgun/views/host_new.py

```diff
@@ -795,6 +795,26 @@ class EditSystemPurposeView(View):
     cancel = OUIAButton('cancel-syspurpose')
 
 
+class ManageHostStatusesView(View):
+    """Manage host statuses modal"""
+
+    ROOT = './/div[@data-ouia-component-id="statuses-modal"]'
+
+    host_statuses_table = PatternflyTable(
+        component_id='statuses-table',
+        column_widgets={
+            'Name': Text('.//td[1]'),
+            'Status': Text('.//td[2]'),
+            'Reported at': Text('.//td[3]'),
+            3: Button(locator='.//td[4]'),
```

Comment:
I would suggest using something like this, instead of indexing it if possible :)
```suggestion
'Name': Text('.//td[contains(@data-label, "Name")]')
```

Reply:
Should be possible, I just wanted to keep the code consistent through the file -- the same indexing is used in other views in the file, e.g. https://github.com/SatelliteQE/airgun/blob/master/airgun/views/host_new.py#L693C2-L701C15. Depends on what's more important

Reply:
yeah, I agree. It's fine if it's not consistent with the rest of the code in the file.

Reply:
Updated PR to address comment. Looks better now!